Assets/Prefabs/Enemies/SalivaBullet.prefab

```diff
@@ -0,0 +1,114 @@
+%YAML 1.1
```

Comment:
is there any other enemies who use this saliva bullet?, if so then leave the name if not rename to AdultSpaceFlyBullet or something along those lines